dangerfile.ts

```diff
@@ -62,7 +62,7 @@ function createComparisonFilter(parsedThreshold: number, gzipThreshold: number)
  */
 function isPackageComparison(comparisonEntry: [string, any]) {
```

Comment:
Do we even need this file in the first place? It feels like

```suggestion
```

could do the job.

Reply:
> Do we need this filter in the first place?

As far as I'm concerned, that's up to the core team to decide 🙂. I'm fine with either way.

Reply:
I imagine the mean downside is about the number of diffs it could create. We could workaround it, say:

```diff
diff --git a/dangerfile.ts b/dangerfile.ts
index bd769f141a..aedc4d72d7 100644
--- a/dangerfile.ts
+++ b/dangerfile.ts
@@ -55,16 +55,6 @@ function createComparisonFilter(parsedThreshold: number, gzipThreshold: number)
   };
 }

-/**
- * Checks if the bundle is of a package e.b. `@mui/material` but not
- * `@mui/material/Paper`.
- * @param {[string, any]} comparisonEntry
- */
-function isPackageComparison(comparisonEntry: [string, any]) {
-  const [bundleKey] = comparisonEntry;
-  return !/^@[\w-]+\/[\w-]+\/.+$/.test(bundleKey);
-}
-
 /**
  * Generates a user-readable string from a percentage change.
  * @param {number} change
@@ -160,12 +150,23 @@ async function reportBundleSize() {
   if (anyResultsChanges.length > 0) {
     const importantChanges = mainResults
       .filter(createComparisonFilter(parsedSizeChangeThreshold, gzipSizeChangeThreshold))
-      .filter(isPackageComparison)
+      .sort(([, a], [, b]) => {
+        const aDiff = Math.abs(a.parsed.absoluteDiff) + Math.abs(a.gzip.absoluteDiff);
+        const bDiff = Math.abs(b.parsed.absoluteDiff) + Math.abs(b.gzip.absoluteDiff);
+        return bDiff - aDiff;
+      })
       .map(generateEmphasizedChange);

     // have to guard against empty strings
     if (importantChanges.length > 0) {
-      markdown(importantChanges.join('\n'));
+      let text = importantChanges.slice(0, 9).join('\n');
+
+      if (importantChanges.length > 10) {
+        text += '\n';
+        text += `and ${importantChanges.length - 10} more changes`;
+      }
+
+      markdown(text);
     }

     const details = `## Bundle size report
```

Reply:
Tweaked the implementation a bit (the slice is off by one 🙂) and added to this PR